src/utils/helpers/MappingHelpers.ts

```diff
@@ -58,6 +58,19 @@ export function createLinks(nodes: DependencyNode[], services: Service[]): Depen
 
         return [...links, ...linksWithTypes];
     }, []);
+
```

Comment:
Maybe there's a neat way to remove the additional loop over mappedLinks and somehow include that logic inside of the reduce function?

Reply:
Done